doc/Install-manually.md

```diff
@@ -41,36 +51,59 @@ wget -q https://github.com/tuub/kitodo-mediaserver/releases/download/${MS_VERSIO
 # Download web UI
 wget -q https://github.com/tuub/kitodo-mediaserver/releases/download/${MS_VERSION}/kitodo-mediaserver-ui-${MS_VERSION}.war -O kitodo-mediaserver-ui.war
 
-# Download default configuration file
-wget -q https://raw.githubusercontent.com/tuub/kitodo-mediaserver/${MS_VERSION}/kitodo-mediaserver-core/src/main/resources/config/local.yml -O config/local.yml;
+# Download default local configuration file
+wget -q https://raw.githubusercontent.com/tuub/kitodo-mediaserver/${MS_VERSION}/kitodo-mediaserver-core/src/main/resources/config/local.yml -O local.yml;
 ```
 
 ### Link the Fileserver and UI to Tomcat
 
 ```bash
-ln -s /usr/local/kitodo-mediaserver/kitodo-mediaserver-fileserver.war /usr/local/tomcat/webapps/
+ln -s /usr/local/kitodo-mediaserver/kitodo-mediaserver-fileserver.war /var/lib/tomcat8/webapps/
 # UI is optionally
-ln -s /usr/local/kitodo-mediaserver/kitodo-mediaserver-ui.war /usr/local/tomcat/webapps/
+ln -s /usr/local/kitodo-mediaserver/kitodo-mediaserver-ui.war /var/lib/tomcat8/webapps/
 ```
 
 ### Create working directories
 
 ```bash
-mkdir -p /var/lib/kitodo-mediaserver
-mkdir /var/lib/kitodo-mediaserver/hotfolder
-mkdir /var/lib/kitodo-mediaserver/importing
-mkdir /var/lib/kitodo-mediaserver/import_error
+mkdir -p /var/local/kitodo-mediaserver
+mkdir /var/local/kitodo-mediaserver/hotfolder
+mkdir /var/local/kitodo-mediaserver/importing
+mkdir /var/local/kitodo-mediaserver/import_error
+mkdir /var/local/kitodo-mediaserver/files
+mkdir /var/local/kitodo-mediaserver/logs
```

Comment:
The working directories should harmonize with default.yml - please change the default.yml too